source/system-administrators/activities/delivery/setup-serverless-delivery.rst

```diff
@@ -0,0 +1,390 @@
+   # The URL for the serverless delivery deployer delete URL
+   studio.serverless.delivery.deployer.target.deleteUrl: ${studio.preview.deleteTargetUrl}
+   # The template name for serverless deployer targets
+   studio.serverless.delivery.deployer.target.template: aws-cloudformed-s3
+   # Replace existing target configuration if one exists?
+   studio.serverless.delivery.deployer.target.replace: false
+   # The URL the deployer will use to clone/pull the site's published repo. When the deployer is in a separate node
+   # (because of clustering), this URL should be an SSH/HTTP URL to the load balancer in front of the Studios
+   studio.serverless.delivery.deployer.target.remoteRepoUrl: ${env:CRAFTER_DATA_DIR}/repos/sites/{siteName}/published
+   # The deployer's local path where it will store the clone of the published site. This property is not needed if
+   # the deployer is not the preview deployer, so you can leave an empty string ('') instead
+   studio.serverless.delivery.deployer.target.localRepoPath: ${env:CRAFTER_DATA_DIR}/repos/aws/{siteName}
+   # Parameters for the target template. Please check the deployer template documentation for the possible parameters.
+   # The following parameters will be sent automatically, and you don't need to specify them: env, site_name, replace,
+   # disable_deploy_cron, local_repo_path, repo_url, use_crafter_search
+   studio.serverless.delivery.deployer.target.template.params:
+      # The delivery Elasticsearch endpoint (optional if authoring is using the same one, specified in the ES_URL env variable)
+      elastic_search_url: https://vpc-serverless-test-jpwyav2k43bb4xebdrzldjncbq.us-east-1.es.amazonaws.com
+      aws:
+        # AWS access key (optional if specified through default AWS chain)
+        default_access_key: XXXXXXXXXX
+        # AWS secret key (optional if specified through default AWS chain)
+        default_secret_key: XXXXXXXXXXXXXXXXXXXX
+        cloudformation:
+          # Namespace to use for CloudFormation resources (required when target template is aws-cloudformed-s3)
+          namespace: serverless-test
+          # The domain name of the serverless delivery LB (required when target template is aws-cloudformed-s3)
+          deliveryLBDomainName: serverless-test-lb-1780491458.us-east-1.elb.amazonaws.com
+
+
+----------------------------------------------------
+Step 4: Create the Site in the Authoring Environment
+----------------------------------------------------
+
+#. Login to the Crafter Studio in the authoring environment from your browser.
+#. Click the ``Create Site`` button
+#. Choose the ``Editorial`` blueprint, enter the ``Site Id`` (e.g. ``editorial``), and then review and create.
+#. Go to your AWS console in your browser and on the ``Services`` dropdown search for CloudFormation. You should then 
+   see  the CloudFormation for the site you just created with the status ``CREATE_IN_PROGRESS``. After a several
```

Comment:
Typo? Remove `a` so the sentence goes like: `After several minutes, the ...`

Reply:
Done